libs/klv/src/main/java/org/codice/alliance/libs/klv/ConvertSubpolygonsToEnvelopes.java

```diff
@@ -30,14 +36,22 @@ public void accept(Visitor visitor) {
     @Override
     public Geometry apply(Geometry geometry) {
 
-        if (geometry.getNumGeometries() == 1) {
+        if (geometry == null || geometry.getNumGeometries() <= 1) {
```

Comment:
I think I would have created a new GeometryOperator decorator called something like NotEmptyGeometryOperator and wrap ConvertSubpolygonsToEnvelopes with it in the blueprint.